circle.yml

```diff
@@ -0,0 +1,18 @@
+machine:
+  node:
+    version: v5.4.1
+
+dependencies:
+  override:
+    - npm install
+  cache_directories:
+    - node_modules
+
+test:
+  override:
+    - npm run lint
+    
+deployment:
+  release:
+    commands:
+      - npm run dev
```

Comment:
Is this because you want the styleguide for github pages?

Reply:
Our css I write as postcss with nested, ect ect (not supported by native css). So I need the builded css file
As we don't want dev push built file, circleCI have to make this job

Reply:
Maybe you just can't write into the directory that way and need to write into the [tmp dir](https://circleci.com/docs/2.0/configuration-reference/#example-for-root-key)?

However I'm not sure you still can access them afterwards, the docker will be turned down

Reply:
Proposal: Having a _dist_ or _prod_ github branch that hosts our files.

Reply:
and with this way Neomad could be call the prod one!

Reply:
considering a `dist` branch, neomad.org could import directly `https://www.gitcdn.xyz/repo/neomad-team/styleguide/dist/content/styles/main.css`